### PR TITLE
Add tomcat-catalina-jmx-remote in Tomcat dependencies

### PR DESCRIPTION
**Pull request by blasd**
Add missing artifactId:
In jetty: javax-websocket-client-impl
in Tomcat: tomcat-catalina-jmx-remote

### CODE REVIEW

File: spring-boot-dependencies/pom.xml [hunk 1]
```diff
@@ -1295,6 +1295,11 @@
 				<version>${tomcat.version}</version>
 			</dependency>
 			<dependency>
+				<groupId>org.apache.tomcat</groupId>
+				<artifactId>tomcat-catalina-jmx-remote</artifactId>
```
Comment (snicoll) [inline spring-boot-dependencies/pom.xml]: Dependencies in this file are ordered. Could you please move it at the right place?

File: spring-boot-dependencies/pom.xml [hunk 2]
```diff
@@ -1647,6 +1652,11 @@
 			</dependency>
 			<dependency>
 				<groupId>org.eclipse.jetty.websocket</groupId>
+				<artifactId>javax-websocket-client-impl</artifactId>
```
Comment (snicoll) [inline spring-boot-dependencies/pom.xml]: Ditto

**Comment by snicoll** [comment]
What is the use case here? That would be the first catalina jar we would add and I don't know if we should do that. Either we add all tomcat jars or we restrict to the embedded case. It looks like we went with the latter for now. 

I am sure @wilkinsona knows more about the history

**Comment by snicoll** [comment]
@blasd  that branch is a bit weird. Can you please rebase cleanly from 1.5.x with a single commit and push force on your branch? Also I've noticed an = sign in your name:

```Author: =Benoit Lacelle <=```

**Comment by blasd** [comment]
@snicoll I use this to enable remote JMX with SpringBoot. Seeing we were manually set the version in our project, I thought it deserved to be moved to spring-boot.
 
https://stackoverflow.com/questions/29412072/how-to-access-spring-boot-jmx-remotely

I may update the PR with all catalina dependencies if it appeared relevant.

**Comment by blasd** [comment]
I open https://github.com/spring-projects/spring-boot/pull/10045 which fix the log history and author (thanks for this).

**Comment by snicoll** [comment]
Duplicate of #10045

**Pull request closed without merging**